bin/blocklock-agent/examples/blocklock/arguments_parser.rs

```diff
@@ -123,6 +123,17 @@ pub struct BlockchainArgs {
     #[arg(long, env = "BLOCKLOCK_SYNC_BATCH_SIZE", default_value = "20")]
     pub sync_batch_size: usize,
 
+    /// How often to synchronize the current state against the chain
+    /// This parameter has an impact on the fulfillment delay of blocklock requests. With 30 seconds,
+    /// blocklock requests will be fulfilled on average 30/2 = 15 seconds after the requested block
+    /// height is reached.
+    #[arg(long, env = "BLOCKLOCK_CONTRACT_SYNC_INTERVAL", default_value = "30")]
```

Comment:
Why 30 as a default? Is this done with Filecoin in mind?

Reply:
Mostly to heavily lower the RPC usage so that we fetch the current block at most once every 30 seconds. 
We can change the default if you guys think it's too much.